internal/cli/cli.go

```diff
@@ -197,18 +210,19 @@ func (c *cli) prepareTenant(ctx context.Context) (Tenant, error) {
 			if err != nil {
 				return Tenant{}, err
 			}
-		} else {
-			// persist the updated tenant with renewed access token
-			t.AccessToken = res.AccessToken
-			t.ExpiresAt = time.Now().Add(
-				time.Duration(res.ExpiresIn) * time.Second,
-			)
+			return t, nil
+		}
 
-			err = c.addTenant(t)
-			if err != nil {
-				return Tenant{}, err
-			}
+		t.AccessToken = res.AccessToken
+		t.ExpiresAt = time.Now().Add(
+			time.Duration(res.ExpiresIn) * time.Second,
+		)
+
+		err = c.addTenant(t)
+		if err != nil {
+			return Tenant{}, err
 		}
+
```

Comment:
Similar to the above, this is a stylistic change to eliminate the `else` in favor of an early return.